src/style.css

```diff
@@ -1 +1,100 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src:
+  local("Avenir"),
+  url(fonts/Avenir.ttc),
+  url(fonts/Avenir-Book.ttf),
+  url(fonts/Avenir-Heavy.ttf);
+}
+
+.search {
+  display: flex;
+  border-radius: 4px;
+  border: 1px solid #E1E7ED;
+  background: #FFF;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.10);
+  box-sizing: border-box;
```

Comment:
It's better practice to move box-sizing to the *{} block
`*{ box-sizing: border-box; }`